ConsoleProject/src/view/Frontend.cs

```diff
@@ -156,18 +178,20 @@ public class Frontend {
                 imageListBox.SelectedIndex = 0;
             }
             DisplayImage();
+            form.Text = AppName + " - " + Path.GetFileName(filename);
```

Comment:
I don't see why is this info required.

Reply:
Just for conveniency of knowing which file is currently being loaded.
Helped me a couple of times when trying multiple repackings with the same images and a repack wouldn't correctly load (without displaying any errors). But I understand where this is coming from, and there should be an error message displayed at some point rather than doing this.

However, most apps that open files do display the filename in the title bar (notepad, paint, etc.)